script/deploy/healthCheck.ts

```diff
@@ -266,32 +270,46 @@ const main = defineCommand({
         ]),
         client: publicClient,
       })
+
       const approvedDexs = await dexManager.read.approvedDexs()
 
-      // Loop through DEXs excluding the address for FeeCollector, LiFiDEXAggregator and TokenWrapper
       let numMissing = 0
-      for (const dex of dexs.filter(
-        (d) => !corePeriphery.includes(getAddress(d))
-      )) {
-        if (!approvedDexs.includes(getAddress(dex))) {
-          logError(`DEX ${dex} not approved in Diamond`)
-          numMissing++
+
+      // Check for each address in dexs.json if it is whitelisted
```

Comment:
great that you noticed it and improved healthCheck script. 

I also noticed that each time I review dexs.json it includes contracts beyond just DEXs and I am confused. Could we rename `dexs.json` to something more general like `whitelisted.json`? That way, we can also remove comments like // Ensure that periphery contracts which are used like DEXs are whitelisted.

Reply:
I totally feel you and second that. 
My plan was to do that when we revise the whole DEXs/Sigs handling (cause also they are not signatures but function selectors).